controllers/admin/AdminStatusesController.php

```diff
@@ -348,7 +348,7 @@ public function renderForm()
                     'name' => 'send_email',
                     'values' => [
                         'query' => [
-                            ['id' => 'on', 'name' => $this->trans('Send an email to the customer when his/her order status has changed.', [], 'Admin.Shopparameters.Feature'), 'val' => '1'],
+                            ['id' => 'on', 'name' => $this->trans('Send an email to the customer when their order status has changed.', [], 'Admin.Shopparameters.Feature'), 'val' => '1'],
```

Comment:
```suggestion
                            ['id' => 'on', 'name' => $this->trans('Send an email to the customer when the order status has changed.', [], 'Admin.Shopparameters.Feature'), 'val' => '1'],
```
Clear enough and grammatically correct 😉